packages/manager/apps/web/client/app/hosting/database/private-sql-activation/private-sql-activation.routing.js

```diff
@@ -0,0 +1,28 @@
+import map from 'lodash/map';
+import sortBy from 'lodash/sortBy';
+import toLower from 'lodash/toLower';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('app.hosting.private-sql-activation', {
+    url: '/private-sql-activation',
+    component: 'privateSqlActivation',
```

Comment:
Maybe it could be renamed like this:
```suggestion
    component: 'hostingDatabasePrivateSqlActivation',
```